HelperFunctions/PeakFitting/voigtfit.m

```diff
@@ -0,0 +1,41 @@
+function [f,gof,output] = voigtfit(x, y, n, init, limits)
+% VOIGTFIT Fit N voigt functions to x and y data
+%   Apply voigtN model to data given an initial guess
+%   INPUT:
+%   x - vector of x-data (same shape as "fit" takes)
+%   y - vector of y-data (same shape as "fit" takes)
+%   n - number (N) of peaks to fit
+%   init - struct with fields (each are Nx1 except background which is 1x1):
```

Comment:
we don't want init to possibly contain eta?

Reply:
I think this is good for an initial commit. Plus, it's hard to get an (incorrect) local maxima for eta.

Reply:
not sure what you mean, but providing a way for the user to set a parameter seems important for a fit function!